Assets/Data/Cards/pawns/romanLegion.asset

```diff
@@ -0,0 +1,28 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: bef7bbcd37c75064dad81e5f407169e2, type: 3}
+  m_Name: romanLegion
+  m_EditorClassIdentifier: 
+  cardArt: {fileID: 0}
+  cardName: Roman Legion
```

Comment:
Is this thematically consistent?

Reply:
welll.... I saw it as the gods manipulating the world including ours but it likely would depend on what the other decks might look like